src/core/internal/traits.d

```diff
@@ -294,6 +294,25 @@ template Fields(T)
         alias Fields = TypeTuple!T;
 }
 
+// std.traits.hasElaborateMove
+template hasElaborateMove(S)
```

Comment:
Please add a complete DDoc header to document this for other developers.

Reply:
The other traits in this module have no ddoc either, but they are all documented on the Phobos side (see https://github.com/dlang/phobos/pull/7075/files#diff-3de50c374a825bac70efe9457fa16e85R3795 for this one), presumably because this is an internal module. AFAICT docs aren't generated for `core.internal`, so using `$(REF ...)` wouldn't really make a difference.

Reply:
Actually, `hasElaborateMove` should go into `core.internal.traits` and `__move_post_blt` should go somewhere in `rt`, but we haven't yet established a strong precedent for that yet.  I don't care if that's done in this PR;  I will probably do it later in a followup if this PR is merged.

The documentation for `rt` is generated for the "Internal API" here:  https://dlang.org/phobos/index.html  The documentation there isn't really for the user; it's for other developers.  Even if the documentation is not published on the website, those developers who utilize a modern editor can see the DDoc comments at each usage of the function, and that makes it quite convenient to browse and understand the code, so that is the motivation for documenting the implementations in this PR.

What was done in the past is not always the best practice to follow.  Some of us are trying to raise the bar for any new code submissions.  Please help the next person that has to maintain this code by documenting the interface.  You can see other contributors are doing this recently with the array runtime hooks, and that would be the preferred precedent to follow.

https://github.com/dlang/druntime/pull/2655/files
https://github.com/dlang/druntime/pull/2648/files
https://github.com/dlang/druntime/pull/2632/files

Reply:
> The documentation there isn't really for the user; it's for other developers. Even if the documentation is not published on the website, those developers who utilize a modern editor can see the DDoc comments at each usage of the function, and that makes it quite convenient to browse and understand the code, so that is the motivation for documenting the implementations in this PR.

That's fair. I've added a `$(REF ...)`.